notebooks/GroupCentrality.ipynb

```diff
@@ -0,0 +1,289 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Group Betweenness"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This class constructs the approximate group betweenness for a given undirected graph G.\n",
+    "\n",
+    "The constructor [ApproxGroupBetweenness(G, groupSize, epsilon)](https://networkit.github.io/dev-docs/python_api/centrality.html?highlight=approxgr#networkit.centrality.ApproxGroupBetweenness) expects an undirected graph, the  desired group size `groupSize`, and the accuracy of the approximation in `epsilon`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Initalize algorithm\n",
+    "btwn = nk.centrality.ApproxGroupBetweenness(G, 10, 0.1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Run \n",
+    "btwn.run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After running the algorithm, we can extract some information about the group betweenness centrality, e.g. the group with the highest approximate betweenness centrality score. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Group with highest betweenness centrality\n",
+    "btwn.groupMaxBetweenness()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One may also have a group of nodes and is interested in the group's betweeness centrality score. The function [scoreOfGroup(group, normalized=False)]() returns the betweenness centrality score of the list of nodes in `group`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Create a random group of nodes\n",
+    "group = [7, 8, 11, 20]\n",
+    "# Get betweenness centrality score of group\n",
+    "btwn.scoreOfGroup(group)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Group Closeness"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This class finds the group of nodes with highest group closeness centrality. The algorithm is the one proposed in Bergamini et al., ALENEX 2018.\n",
+    "\n",
+    "The constructor [GroupCloseness(G, k=1, H=0)](https://networkit.github.io/dev-docs/python_api/centrality.html?highlight=group#networkit.centrality.GroupCloseness) expects an unweighted graph and the desired group size in the parameter `k`. If `H` is equal to 0, simply runs the algorithm proposed in Bergamini et al.. If `H` > 0, interrupts all BFSs after `H` iterations. For very large networks, it is suggested to set `H` to a reasonable value."
```

Comment:
```suggestion
    "The constructor [GroupCloseness(G, k=1, H=0)](https://networkit.github.io/dev-docs/python_api/centrality.html?highlight=group#networkit.centrality.GroupCloseness) expects an unweighted graph and the desired group size in the parameter `k`. If `H` is equal to 0, simply runs the algorithm proposed in Bergamini et al.. If `H` > 0, all BFSs will explore the graph up to distance `H`. This can speed up the algorithm, at the cost of a lower solution quality."
```